### ISSUE TITLE
Docs say to add your settings to mycroft.ini which doesn't exist.

### ISSUE THREAD

**Issue by klundry** [forslund/mpd_skill#10]
It seems the configuration file type and format has changed.  It says to add 
  [MPDSkill]
  mpd_url=YOUR_URL
  mpd_port=YOUR_PORT

to mycroft.ini but that file doesn't exist. I think mycroft.conf is the relevant file now but it has a different format. Is there an example of how to add my settings to mycroft.conf?

**Comment by forslund** [comment]
Thank you for reporting. This one has fallen a bit behind, I'll update the section...

A quick response is to create a mycorft.conf in ~/.mycroft/ (if it doesn't exist) and make it look something like

```json
{
  "MPDSkill": {
    "mpd_url": "YOUR_URL",
    "mpd_port": "YOUR_PORT"
  }
}
```

In the long run we should move to skill settings and using mycroft home to set the ip

**Comment by Upperholme** [comment]
And if MPD requires a password, can I include that in the conf file also, perhaps in the format:

`{
  "MPDSkill": {
    "mpd_url": "YOUR_URL",
    "mpd_port": "YOUR_PORT",
   "mpd_passwd": "PASSWORD"
  }
}`
?

**Comment by Upperholme** [comment]
I have my mycroft.conf file set up:
`{
  "MPDSkill": {
    "mpd_url": "IP address of mpd server",
    "mpd_port": "6600"
  }
}`
mycroft log reports: 

`mpd - INFO - Calling MPD connect:('localhost', 6600, timeout=None)
mpd_skill__init__:_connect:100 - INFO - Could not connect to server, retrying in 10 sec`

So it looks like the MPD skill isn't reading ~/.mycroft/mycroft.conf as it appears to be trying to load a default config

Any pointers most welcome.

**Comment by klundry** [comment]
Yeah I haven't been able to get it to work either. I guess we'll have to hope the skill gets updated at some point.

**Comment by forslund** [comment]
It will :)

I'm moving it over to use skill settings instead of the configuration, but I got side tracked...Will make an effort to get the final pieces in place and make sure it's compatible with the latest changes in mycroft-core.

**Comment by klundry** [comment]
Awesome!

**Comment by Upperholme** [comment]
Many thanks.

**Comment by vgnmnky** [comment]
Sorry to resurrect this, but I don't know if it was fixed.  I've added my MPD IP address to .mycroft/mycroft.conf, but it still tries to connect to localhost.  Am I missing something?  I've looked elsewhere for settings, but I can't find anything.  Thank you.

**Comment by klundry** [comment]
> Sorry to resurrect this, but I don't know if it was fixed. I've added my MPD IP address to .mycroft/mycroft.conf, but it still tries to connect to localhost. Am I missing something? I've looked elsewhere for settings, but I can't find anything. Thank you.

I don't think it was ever updated to work with the new way of doing skill settings on home.mycroft.ai. I haven't tried it in a while but it doesn't show up on the skills marketplace. It should be possible to add it manually but I don't know if it will work.

**Comment by forslund** [comment]
Gah, I'm terrible with docs. Today I pyshed the change moving it over to using the skill settings through home.mycroft.ai

**Comment by forslund** [comment]
It may still need a restart after setting have been changed on home for it to take effect. let me know if that isn't working then I've done something wrong

**Comment by klundry** [comment]
Excellent! Any idea how long it takes for the skill to get into the marketplace? I'm not sure how that works. Is that something you have to submit?

Looking forward to trying it out.

**Comment by forslund** [comment]
This hasn't been submitted yet since it's been a bit outdated...during the christmas holiday I got some time to push it into a usable (for some definition of usable) state again

**Comment by klundry** [comment]
OK, so would you recommend installing it via the github repo like this? "msm install https://github.com/forslund/mpd_skill.git"?

**Comment by forslund** [comment]
That's the way I install it since it automatically install the mpd dependency into the mycroft virtual environment.

**Comment by klundry** [comment]
I had to restart but it is working! Great!

I can't get it to play any if my playlists though. Not sure if there is a specific way to ask that or if its a bug or unimplemented feature.
